mathics/builtin/quantum_mechanics/angular.py

```diff
@@ -0,0 +1,220 @@
+"""
+Angular Momentum
+
+<url>:Angular momentum: https://en.wikipedia.org/wiki/Angular_momentum</url> in physics is the rotational analog of linear momentum. It is an important quantity in physics because it is a conserved quantity the total angular momentum of a closed system remains constant.
+"""
+
+from typing import List
+
+from sympy.physics.matrices import msigma
+from sympy.physics.quantum.cg import CG
+from sympy.physics.wigner import wigner_3j, wigner_6j
+
+from mathics.builtin.base import SympyFunction
+from mathics.core.atoms import Integer
+from mathics.core.convert.sympy import from_sympy
+from mathics.core.evaluation import Evaluation
+from mathics.core.list import ListExpression
+from mathics.core.symbols import Symbol
+
+from mathics.core.attributes import (
+    listable as A_LISTABLE,
+    # numeric_function as A_NUMERIC_FUNCTION,
+    protected as A_PROTECTED,
+    read_protected as A_READ_PROTECTED,
+)
+
+
+class ClebschGordan(SympyFunction):
+    """
+    <url>:Clebsch-Gordan coefficients matrices: https://en.wikipedia.org/wiki/Clebsch%E2%80%93Gordan_coefficients</url> (<url>:SymPy: https://docs.sympy.org/latest/modules/physics/quantum/cg.html</url>, <url>:WMA: https://reference.wolfram.com/language/ref/ClebschGordan</url>)
+
+    <dl>
+      <dt>'ClebschGordan[{$j1$, $m1$}, {$j2$, $m2$}, {$j$ $m$}]'
+      <dd>returns the Clebsch-Gordan coefficient for the decomposition of in terms of |$j$, $m$|$j2$, $m2$>.
```

Comment:
To be precise,   * ... in terms of |$j_1, m_1$>|j_2 m_2$>.*

Maybe $\langle$ instead of $>$.

Reply:
The persistent problem we have is that we the homegrown doc system really can't handle symbols reliably. We have seen this time and time again with things like Phi. 

Do we use the ket symbols anywhere else? If you know that when we go to print this it will work, then we can use.

Reply:
OK, let's keep ">", but then the description should read
*returns the Clebsch-Gordan coefficient for the decomposition of in terms of $|j_1, m_1>|j_2, m_2>$.*

Reply:
Thanks for finding the mistake. I think it is fixed now in bb73c516

Reply:
Speaking of >, the ket symbol and our homegrown docsystem fragility. When I was working on the guide section for the process of adding Curl as an example,  I wanted to write:

$$     ( \partial f_3 / \partial x_2 - \partial f_2 / \partial x_3, \ \ %
      \partial f_1 / \partial x_3 - \partial f_3 / \partial x_1, \ \ %
      \partial f_2 / \partial x_1 - \partial f_1 / \partial x_2 )
$$

in the Sphinx document. and I was pleasantly surprised that I could do this without fuss, and that there was a world of flexibility available rather that this kind of constrained system where we are working around things and getting less-good results in the end.

So there _is_ hope if we can get over the mountain of moving this to sphinx.